core/transcoder.go

```diff
@@ -31,14 +31,15 @@ func (lt *LocalTranscoder) Transcode(fname string, profiles []ffmpeg.VideoProfil
 	}
 	defer os.RemoveAll(fullDirName)
 	tr := transcoder.NewFFMpegSegmentTranscoder(profiles, fullDirName)
-	mid, seqNo, parseErr := parseURI(fname)
+	_, seqNo, parseErr := parseURI(fname)
 	start := time.Now()
-	if monitor.Enabled && parseErr == nil {
-		monitor.LogSegmentTranscodeStarting(seqNo, mid)
-	}
 	data, err := tr.Transcode(fname)
 	if monitor.Enabled && parseErr == nil {
-		monitor.LogSegmentTranscodeEnded(seqNo, mid, time.Since(start), common.ProfilesNames(profiles))
+		// This will run only when fname is actual URL and contains seqNo in it.
+		// When orchestrator works as transcoder, `fname` will be relative path to file in local
+		// filesystem and will not contain seqNo in it. For that case `SegmentTranscoded` will
```

Comment:
Not of course, and will need to be changed in case of changes like in #846, but it's best what can be done now - because for local files here we only have random name and no other information.